web/src/typing_data.ts

```diff
@@ -64,6 +65,25 @@ export function get_topic_typists(stream_id: number, topic: string): number[] {
     return muted_users.filter_muted_user_ids(typists);
 }
 
+export function add_edit_message_typing_id(message_id: number): void {
+    if (!edit_message_typing_ids.includes(message_id)) {
+        edit_message_typing_ids.push(message_id);
+    }
+}
```

Comment:
Don't use a list data structure, please use a Map or a Set. Efficiency doesn't matter here as one is unlikely to have dozens of these, but one should always use correct data structure types